### PR TITLE
#38/🐣: fix of the css structure, button removing, deleting usefull ti…

### PR DESCRIPTION
**Pull request by elkinny**
**What was done**
fix of the css structure, button removing, deleting usefull time classes on the sky and building

**Issue was fixed**
#39

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
             <div class="building"></div>
             <div class="ground"></div>
         </div>
-        <div class="sky hidden"></div>
```
Comment (elkinny) [inline src/index.html]: OMG SERIOUSLY, HOW COULD I?! Sorry senpai!

File: src/js/sky/sky.js [hunk 2]
```diff
@@ -11,13 +12,12 @@ export default class Sky {
         const sky = document.createDocumentFragment();
         const cloud = document.createElement('div');
 
-        cloud.classList.add('cloud', ...this.mode);
```
Comment (elkinny) [inline src/js/sky/sky.js]: I told you to remove it!
You promised!!!!!
plz don't be angry

File: src/index.html (hunk 1 above)
Comment (Dmitry-White) [reply src/index.html]: Alright, alright, child, don't cry. Wisdom comes with experience!

File: src/js/sky/sky.js (hunk 2 above)
Comment (Dmitry-White) [reply src/js/sky/sky.js]: AHRRRRRRR!

**Pull request merged**